### ISSUE TITLE
How should we show multiple journeys?

### ISSUE THREAD

**Issue by RaitoBezarius** [RaitoBezarius/wannago#10]
It would be interesting to show alternatives to the first journey which we get from the API.

But we need a way to show them without polluting the output.

Any ideas?

**Comment by Shir0kamii** [comment]
I don't think we have to do it for the 1.0 version. It should be added in a later release, and we must concentrate on essential features.

**Comment by RaitoBezarius** [comment]
Ok, let's plan it for later then.
